Extensions/callback-data.md

```diff
@@ -38,12 +40,64 @@ X-Cinode-User-Id: 123
 {}
 ```
 
+Subsequent requests will include a `filters` property with values to filter the returned data.
+
+```http
+POST /api/data HTTP/1.1
+Content-Type: application/vnd.cinode.callback.v1+json
+Digest: sha-256=...
+X-Cinode-Signature: ...
+X-Cinode-Company-Id: 123
+X-Cinode-User-Id: 123
+
+{
+    "filters": {
+        "recipient": {
+            "value": "asdasdasdasdasdas"
+        },
+        "some-complex-query": {
+            "value": "my query value"
+        },
+        "status": {
+            "value": ["new", "completed"]
+        }
+    }
+}
+```
+
+Response must contain a `properties`, and `values` properties. Optionally a `filters` and/or `pagination` property.
+
 ```http
 HTTP/1.1 200 OK
 Content-Type: application/json
 
 {
     "data": {
+        // Map of form elements to define filters and facets
+        "filter": {
+            "recipient": {
+                "$type": "input",
+                "label": {"en": "Recipiant name"},
+            },
+            "some-complex-query": {
+                "$type": "input",
+                "label": {"en": "Query"},
+            },
+            "status": {
+                "$type": "multiselect",
+                "label": {"en": "Satus"},
+                "values": [
```

Comment:
Afaik they wanted dynamic values (?)

something like 
```
"options": {
    $type: "static" | "dynamic",
    values: [{"value": "new", "label": "New"}] | dispatchUrl
}
```

Reply:
The values are dynamic since each fetch of the data source would populate the filter's values